pkg/config/testdata/tnf_test_config.yml

```diff
@@ -29,6 +29,9 @@ certifiedcontainerinfo:
 certifiedoperatorinfo:
   - name: etcd-operator
     organization: redhat-marketplace
+acceptedTaints:
```

Comment:
Maybe we should rename it to acceptedKernelTaints, to avoid confusion with Kubernetes node taints.

Reply:
Agreed.  I can change this.